### ISSUE TITLE
The new configuration for startLines in tag-lines is reported as invalid by the schema validation

### ISSUE THREAD

**Issue by stof** [gajus/eslint-plugin-jsdoc#1021]
## Expected behavior

The config should be accepted as valid

## Actual behavior

Eslit fails with this output:

```
Oops! Something went wrong! :(

ESLint: 8.38.0

Error: .eslintrc.json:
	Configuration for rule "jsdoc/tag-lines" is invalid:
	Value {"startLines":1} should be string.

```

## ESLint Config

```json
{
    "plugins": ["jsdoc"],
    "rules": {
        "jsdoc/tag-lines": ["error", {
            "startLines": 1
        }],
    }
}
```

## Environment

- Node version: 18.16.0
- ESLint version 8.38.0
- `eslint-plugin-jsdoc` version: 43.0

**Comment by brettz9** [comment]
The first argument should be "any", "always", or "never".  Closing as that should resolve, but feel free to comment further as needed.

**Comment by stof** [comment]
The readme does not show any example of configuring this rule. the rule is there in the TOC but the anchor does not work.

**Comment by brettz9** [comment]
Yes, apologies, that is related to #886. We really need a fix on our docs, @gajus , as well as the release notes.

**Comment by stof** [comment]
For reference to anyone finding this issue, the working configuration looks like that:

```json
{
    "plugins": ["jsdoc"],
    "rules": {
        "jsdoc/tag-lines": ["error", "never", {
            "startLines": 1
        }],
    }
}
```

**Comment by brettz9** [comment]
Thanks for that. Note that if you are using `sort-tags`, you should use "any" instead of "never".
